backend/routers/v1/inventory.py

```diff
@@ -9,19 +9,21 @@
 
 All routes are prefixed with /api/v1/inventory
 """
-from fastapi import APIRouter
+from fastapi import APIRouter, Response
 from models.snack import (
     Snack,
     SnackCreateSchema,
-    SnackUpdateSchema
+    SnackUpdateSchema,
+    BulkSnackCreate,
+    BulkSnackResponse
 )
 from models.inventory import InventoryResponse
 from utils.db import (
     get_inventory, 
     get_snack, 
     delete_snack,
     create_snack,
-    update_snack
+    update_snack,create_bulk_items
```

Comment:
lets make this formatting uniform
```suggestion
    update_snack,
    create_bulk_items